lui.el

```diff
@@ -1063,6 +1068,8 @@ unexpecting user.")
        (if (> lui-input-marker (point))
            (delete-region (point) lui-input-marker)
          (set-marker lui-input-marker (point)))
+       (let ((inhibit-read-only t))
+         (add-face-text-property lui-output-marker lui-input-marker 'lui-prompt-face))
        (add-text-properties lui-output-marker lui-input-marker
```

Comment:
Huh? Why does this need inhibit-read-only, but the add-text-properties field does not?

Reply:
I don't know but it seems necessary.